sos/report/plugins/kernel.py

```diff
@@ -136,7 +136,8 @@ def setup(self):
             "/sys/kernel/debug/extfrag/extfrag_index",
             clocksource_path + "available_clocksource",
             clocksource_path + "current_clocksource",
-            "/proc/pressure/"
+            "/proc/pressure/",
+            "/boot/config-%s" % self.policy.kernel_version()
```

Comment:
Please use an f-string here as we are converting the project to use those globally.

Reply:
For sure! Can I also change [line 114](https://github.com/sosreport/sos/pull/3580/files#diff-622aa52be019af33ea587ea315ab1be1333530c98581290b57c6e44e493bae2cR114) to use an f string as well?

Reply:
> For sure! Can I also change [line 114](https://github.com/sosreport/sos/pull/3580/files#diff-622aa52be019af33ea587ea315ab1be1333530c98581290b57c6e44e493bae2cR114) to use an f string as well?

you sure can :)

Reply:
Updated to use f-strings! :)